cpp/src/benchmark_core.h

```diff
@@ -297,8 +295,8 @@ void bulk_evict_benchmark(Aggregate agg, Experiment exp) {
         for (i = start_pos; i < stop_pos; /* nop */) {
             std::atomic_thread_fence(std::memory_order_seq_cst);
 
-            agg.bulkEvict(i-1);
-            for (typename Aggregate::timeT j = 0; j < exp.window_size && i < stop_pos; ++j, ++i) {
+            agg.bulkEvict(i - exp.window_size + exp.bulk_size - 1);
```

Comment:
That looks correct to me, based on the following example math:
```
window_size = 10
current window contents [0,1,2,3,4,5,6,7,8,9]
i = 10 # next timestamp to insert
bulk_size = 3
evict up to (i - window_size + bulk_size - 1) = 10 - 10 + 3 - 1 = 2
window contents after bulk evict [3,4,5,6,7,8,9]
```

Reply:
This looks good to me.